sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -915,21 +915,21 @@ class EthService(
   }
 
   @RpcMethod("zen_getForwardTransfers")
-  def getForwardTransfers(hash: Hash): ForwardTransfersView = {
+  def getForwardTransfers(tag: String): ForwardTransfersView = {
     applyOnAccountView { nodeView =>
       nodeView.history
-        .getStorageBlockById(bytesToId(hash.toBytes))
+        .getStorageBlockById(getBlockIdByHashOrTag(nodeView, tag))
```

Comment:
actually we don't want to use `tags`, only hash or height
same for zen_getFeePayments

Reply:
Done